### PR TITLE
Add GUI to navigation.

### PR DESCRIPTION
**Pull request by evilnick**
Minor updates to controller

@matthewhelmke

### CODE REVIEW

File: src/navigation.tpl [hunk 1]
```diff
@@ -23,7 +23,14 @@
                 <li><a href="help-rackspace.html">Rackspace</a></li>
             </ul>
         </li>
-        <li><a href="controllers.html">Controllers</a></li>
+        <li class="section><a href="controllers.html">Controllers</a>
+            <ul class="sub">
+                <li><a href="controllers-creating.html">Creating/Bootstrap</a></li>
```
Comment (matthewhelmke) [inline src/navigation.tpl]: We have a gerund/imperative combo here with Creating/Bootstrap. Should we pick one form and use it for both? Either "Create/Bootstrap" or "Creating/Bootstrapping"?

**Comment by matthewhelmke** [comment]
Only one comment. Otherwise, LGTM.
:v:

File: src/navigation.tpl (hunk 1 above)
Comment (evilnick) [inline src/navigation.tpl]: yeah, I hate 'bootstrapping'. fair enough, I will change it to Create

**Comment by evilnick** [comment]
fixed. thx!

**Pull request merged**